manifest.json

```diff
@@ -1,19 +1,22 @@
 {
+  "manifest_version": 2,
+  "author": "Soreine & Zhouzi",
+  "homepage": "https://soreine.github.io/hyper-gwent",
   "name": "Hyper Gwent",
-  "description": "Enrich pages containing GwentⓇ related content. Card names quick access, terms highlight and more!",
+  "description": "Show tooltips for cards in pages containing GwentⓇ related content.",
```

Comment:
Not sure if that's an issue but this description is different than the one on the website.